tests/cases/conformance/jsdoc/jsdocTypesInTypeAnnotations.ts

```diff
@@ -0,0 +1,22 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @module: commonjs
+// @filename: node.d.ts
+// @noImplicitAny: true
+declare function require(id: string): any;
```

Comment:
Actually, I don't think this test is testing anything unique. I'll just remove it.